legend-engine-server/pom.xml

```diff
@@ -95,6 +95,10 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-executionPlan-execution-store-relational-connection-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-executionPlan-execution-store-authentication</artifactId>
```

Comment:
Don't think authentication falls under a store. But looks like the module is already present

Reply:
Yes. For now the authn module is tied to store specific constructs like DataSourceSpecification etc,